content/style_guide.md

```diff
@@ -565,14 +587,89 @@ like "Create a new user" (by its very nature, a user created **is** a
 new user) or (for a setting named `cidr_block`) "The CIDR block for the
 VPC."
 
+## Examples
+
+### Example Domains
+
+Use the `example.com` domain for generic domains and email addresses in the documentation. Use the `chef.io` or `progress.com` domain for examples that should refer back to Chef Software.
+
+### Example Names
+
+Don't reveal personal information in examples, such as the names of real people, real email addresses, or phone numbers.
+
+Don't use the names of bands, musicians, or characters from works that are under copyright.
+
+When writing about security, follow the accepted convention and use "Alice" and "Bob". Following this convention helps readers see that they are reading a topic about security and integrate the Chef information with their existing knowledge.
+
+Here is a list of some example names for you to use (all of the last names are translations of "Chef", according to the internet):
```

Comment:
I'm not sure I understand what "all of the last names are translations of "Chef", according to the internet" means, but ok 🤷

Reply:
Literally, the magic of Google translate.